src/XMLexport.h

```diff
@@ -44,8 +48,10 @@ class TVar;
 class VarUnit;
 
 
-class XMLexport : public QXmlStreamWriter
+class XMLexport : public QObject
 {
+    Q_OBJECT
```

Comment:
This class does not *seem* to be using signals/slots so does it *need* the `Q_OBJECT` macro?